Gemfile

```diff
@@ -85,6 +85,14 @@ unless packaging?
     gem "guard", require: false
     gem "guard-rspec", require: false
     gem "guard-rubocop", require: false
+    gem "jquery"
```

Comment:
Please remove this. We are using jQuery from the webpack configuration.

Reply:
One thing that could be done is to add the peek css and js in webpack just for development.

Reply:
> One thing that could be done is to add the peek css and js in webpack just for development.

This might be tricky or not even possible because peek's assets relies on rails [0] [1]. For [0] that wouldn't be a problem because it's still a valid comment in js and we can include its dependency. But for [1] I think that might cause an error since it's not valid css. 

If that's the case, I'm fine with having the dedicated assets files to be built just for peek. I would only rename it to `peek` instead of `debug`.

[0] https://github.com/peek/peek/blob/master/app/assets/javascripts/peek.js#L1
[1] https://github.com/peek/peek/blob/master/app/assets/stylesheets/peek.scss#L1

Reply:
Just realized that it's a `scss` file, so the comment will still be valid but we would need to configure webpack to load `scss` files properly.